### DISCUSSION TITLE
How to handle fast click events that can push a configuration to prevent crash?

### DISCUSSION THREAD

**Discussion by plusmobileapps** [badoo/Decompose#267] [Q&A] [answered]
I noticed one thing when I clicked fast on a button that could push a new configuration, but because it was so fast it actually caused a crash I think due to the changes from #244 . 

```
Uncaught Kotlin exception: kotlin.IllegalStateException: Configurations in the stack must be unique
```

Should click events just be throttled of pushing a configuration? Or maybe silently throw an error in a callback if it already exists on the stack?

**Comment by arkivanov** [comment, marked as answer]
This probably because of the animations. The previous screen is still interactive while it is being animated. You can throttle clicks - probably easiest solution. Or disable it, if it is not the current screen.
